### DISCUSSION TITLE
Pi Zero W with OLED Type 3, possible?

### DISCUSSION THREAD

**Discussion by K2Kevin** [sethvoltz/OctoPrint-DisplayPanel#84] [Q&A]
I have a few Pi Zero W used for another purpose previously (Pi-Star Ham Radio Hotspots).  They have an attached OLED Type 3 display.  Will this work on that display panel?  If I have omitted any information, I would be happy to provide.

**Comment by kgutwin** [comment]
It's likely to work, given that those OLED displays seem fairly standardized. The main question will be whether or not the I2C address is the same as the default (`3c`) or if it's connected to a working I2C port on the Pi. 

Try following the first few setup steps from the [README](https://github.com/sethvoltz/OctoPrint-DisplayPanel/blob/main/README.md) and let us know what you find!

**Reply by K2Kevin** [reply]
I did find that it is at 3C I2C.

Update: I did actually go through your instructions previously to confirm, but did it once again now just to be sure.  I think I may have another issue, since plugs seem to crash Octoprint during install.  Just changed Locale to see if that helps.

     0  1  2  3  4  5  6  7  8  9  a  b  c  d  e  f
00:          -- -- -- -- -- -- -- -- -- -- -- -- -- 
10: -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- 
20: -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- 
30: -- -- -- -- -- -- -- -- -- -- -- -- 3c -- -- -- 
40: -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- 
50: -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- 
60: -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- 
70: -- -- -- -- -- -- -- --

**Comment by K2Kevin** [comment]
Found a support post about crashing plugins, and a fix. Changing the locale setting in config.  Did that and IT WORKS!  Thanks for your response!


![IMG_1338](https://user-images.githubusercontent.com/27362912/150202248-4a752b9d-aefb-470a-a115-fa0fe93226b8.jpeg)

**Comment by K2Kevin** [comment]
I do find when flipping display 180, the Virtual Panel (Micro Panel Tab) displays this upside-down.  LOL  So it does in fact mirror the display.  Obviously not what I expected :)  
180 flip more or less makes that Virtual Panel option useless or at least interesting.
